src/app/Plans/PlansPage.tsx

```diff
@@ -75,13 +73,11 @@ const PlansPage: React.FunctionComponent = () => {
                     content={addPlanDisabledObj.content}
                   >
                     <div className={`${spacing.mtMd}`}>
-                      <Button
-                        isDisabled={addPlanDisabledObj.isTooltipEnabled}
-                        onClick={toggleWizard}
-                        variant="primary"
-                      >
-                        Create migration plan
-                      </Button>
+                      <Link to="/plans/create">
+                        <Button isDisabled={addPlanDisabledObj.isTooltipEnabled} variant="primary">
```

Comment:
@gildub I tested rendering this button with `isDisabled={true}` and the Link still takes you to the wizard page. Also, `<Link>` renders an `<a>` and having a `<button>` inside an `<a>` is not semantically allowed in HTML (even though it renders, we're probably getting a warning about it).

Instead we should probably just use the button and navigate in its `onClick` handler. You can use the `useHistory` hook from `react-router-dom` and do `useHistory().push('/plans/create')`. https://reactrouter.com/web/api/Hooks/usehistory

Reply:
Oh I see re having `<button>` embedded in a `<a>`. I was actually in unknown territory for me when I did that this morning. 
I guess I pick up a wrong approach. Good to know, thanks.

Let me use the `useHistory`.